client/lib/plans/features-list.js

```diff
@@ -1129,20 +1131,22 @@ export const FEATURES_LIST = {
 		getSlug: () => constants.FEATURE_PRODUCT_BACKUP_REALTIME_V2,
 		getIcon: () => 'cloud-upload',
 		getTitle: () =>
-			getJetpackCROActiveVersion() === 'v2'
-				? i18n.translate( 'Backup {{strong}}{{em}}Real{{nbh/}}time{{/em}}{{/strong}}', {
-						components: {
-							em: <em />,
-							strong: <strong />,
-							nbh: <>&#8209;</>,
-						},
-						comment: '{{nbh}} represents a non breakable hyphen',
-				  } )
-				: i18n.translate( 'Backup {{em}}Real-time{{/em}}', {
-						components: {
-							em: <em />,
-						},
-				  } ),
+			( {
+				v2: i18n.translate( 'Backup {{strong}}{{em}}Real{{nbh/}}time{{/em}}{{/strong}}', {
+					components: {
+						em: <em />,
+						strong: <strong />,
+						nbh: <>&#8209;</>,
+					},
+					comment: '{{nbh}} represents a non breakable hyphen',
+				} ),
+				i5: i18n.translate( 'Backup Real-time (off-site)' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **25** times:
`translate( 'Real-time Off-site Backups' )` *ES Score: 9*